apps/metadata-editor/src/index.html

```diff
@@ -7,6 +7,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="assets/favicon.ico" />
     <link rel="stylesheet" href="assets/css/materials-symbols-outline.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/gh/lipis/flag-icons@7.2.3/css/flag-icons.min.css"
+    />
```

Comment:
I think we should import this in our CSS and get rid of the external dependency here. If I saw right you've already installed the npm package.